recipe/conda_build_config.yaml

```diff
@@ -70,27 +72,29 @@ flann:
 fontconfig:
   - 2.12
 freetype:
-  - 2.8                # [not ppc64le]
+  - 2.8.1              # [not ppc64le]
 gstreamer:
   - 1.12
 gst_plugins_base:
   - 1.12
+gdal:
+  - 2.2.*
 geos:
   - 3.6.2
 giflib:
   - 5.1
 glib:
-  - 2.53
+  - 2.55
 glpk:
-  - 4.62
+  - 4.61
 gmp:
   - 6
```

Comment:
`pin_the_slow_way` had `6.1.*`; note that 6.1 [added symbols](https://abi-laboratory.pro/tracker/timeline/gmp/).

Reply:
Please see [this (outdated) comment]( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/7#pullrequestreview-98642553 ). FWIU `run_exports` lower bounds the version based on the one used for the build. More details in [this doc]( https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#pin-downstream ).

cc @isuruf